src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -71,15 +78,26 @@ public function load($data, $type = null)
 
             if (null !== $collectionOperations = $resourceMetadata->getCollectionOperations()) {
                 foreach ($collectionOperations as $operationName => $operation) {
-                    $this->addRoute($routeCollection, $resourceClass, $operationName, $operation, $resourceShortName, true);
+                    $this->addRoute($routeCollection, $resourceClass, $operationName, $operation, $resourceShortName, OperationTypes::COLLECTION);
                 }
             }
 
             if (null !== $itemOperations = $resourceMetadata->getItemOperations()) {
                 foreach ($itemOperations as $operationName => $operation) {
-                    $this->addRoute($routeCollection, $resourceClass, $operationName, $operation, $resourceShortName, false);
+                    $this->addRoute($routeCollection, $resourceClass, $operationName, $operation, $resourceShortName, OperationTypes::ITEM);
                 }
             }
+
+            foreach ($this->propertyNameCollectionFactory->create($resourceClass) as $property) {
+                $propertyMetadata = $this->propertyMetadataFactory->create($resourceClass, $property);
+
+                if (!$propertyMetadata->hasNestedOperation()) {
+                    continue;
+                }
+
+                $operation = ['method' => 'GET', 'property' => $property];
+                $this->addRoute($routeCollection, $resourceClass, strtolower($property).'_get', $operation, $resourceShortName, OperationTypes::NESTED);
```

Comment:
I can imagine the route name as such: `api_products_get_images_subcollection`

So `$operationName` should be 'get_'.strtolower($property)`.